keps/sig-apps/2307-job-tracking-without-lingering-pods/README.md

```diff
@@ -305,28 +307,33 @@ the owner reference.
   - Job tracking without lingering Pods
   - Removal of finalizer when feature gate is disabled.
   - Support for [Indexed Jobs](https://git.k8s.io/enhancements/keps/sig-apps/2214-indexed-job)
-- Tests: unit, integration, E2E
-
+- Tests: unit, integration.
 #### Alpha -> Beta Graduation
 
 - Processing 5000 Pods per minute across any number of Jobs, with Pod creation
-  having higher priority than status updates. This might depend on
-  [Priority and Fairness](https://git.k8s.io/enhancements/keps/sig-api-machinery/1040-priority-and-fairness).
+  having higher priority than status updates, using [Priority and Fairness](https://kubernetes.io/docs/concepts/cluster-administration/flow-control).
```

Comment:
@lavalamp this is what we discussed offline. This is the behavior I would like, but maybe this is the wrong approach due to backwards compatibility. Let me know if you have better suggestions.

Reply:
5000/60 = 83 QPS; if each request is 100ms, that's a concurrency of ~9, which is not very much. You probably don't even need to adjust the default APF levels (you do need to adjust or turn off the client side rate limit).

Note that pod creations have a bunch of follow-on requests and there's not much point in giving this special priority to create them if the rest of the system can't then keep up.

What you should do instead is look at the shares given to workload-high and the other PLs and figure out what overall in-flight limit needs to be such that workload high ends up with at least that much concurrency, and document that, so operators know how big of a cluster they need to get that throughput.

Note that to get your maximum throughput, you need to multiplex your requests across all apiservers, which is challenging in some deployments. But it may be more important than e.g. making a new PL.

Reply:
If I determine that we need an update of the existing PLs, should that be (and can it be) dependent on the feature gate for this KEP?

Reply:
That would be very awkward, so I'd recommend no. Not every k8s cluster needs this to go this fast, so I think in that case we'd most likely ask you to just document what needs to be done. But let's see what that is before making any decisions.

Reply:
> I think we are good with the current default APF configuration. And we can recommend users to set the client QPS to 100 if they expect to have 2500 pods finishing per minute or disable it completely if they expect to process more than 5k pods per minute.

SGTM

Reply:
Updated.